plugins/destination/mysql/client/transformer.go

```diff
@@ -0,0 +1,166 @@
+package client
+
+import (
+	"strings"
+
+	"github.com/cloudquery/plugin-sdk/schema"
+)
+
+func (*Client) TransformBool(v *schema.Bool) any {
+	if v.Status != schema.Present {
+		return nil
+	}
+	return v.Bool
+}
+
+func (*Client) TransformBytea(v *schema.Bytea) any {
+	if v.Status != schema.Present {
+		return nil
+	}
+	return v.Bytes
+}
+
+func (*Client) TransformFloat8(v *schema.Float8) any {
+	if v.Status != schema.Present {
+		return nil
+	}
+	return v.Float
+}
+
+func (*Client) TransformInt8(v *schema.Int8) any {
+	if v.Status != schema.Present {
+		return nil
+	}
+	return v.Int
+}
+
+func (*Client) TransformTimestamptz(v *schema.Timestamptz) any {
+	if v.Status != schema.Present {
+		return nil
+	}
+	return v.Time
+}
+
+func (*Client) TransformJSON(v *schema.JSON) any {
+	if v.Status != schema.Present {
+		return nil
+	}
+
+	return string(v.Bytes)
+}
+
+func (*Client) TransformUUID(v *schema.UUID) any {
+	if v.Status != schema.Present {
+		return nil
+	}
+	// We need a slice instead of a fixed sized array
+	bytes := make([]byte, 16)
+	copy(bytes, v.Bytes[:])
+	return bytes
+}
+
+func (*Client) TransformUUIDArray(v *schema.UUIDArray) any {
+	if v.Status != schema.Present {
+		return nil
+	}
+
+	result := make([]string, len(v.Elements))
+	for i, elem := range v.Elements {
+		result[i] = elem.String()
+	}
+
+	return strings.Join(result, ",")
```

Comment:
You need just to use `v.String()` as return value, otherwise the default revere transformer would [fail](https://github.com/cloudquery/plugin-sdk/blob/main/schema/uuid_array.go#L62) on the `{` char absence

Reply:
Same goes for other array types except for bytes